NavigationSystem/Tests/unit-tests/TestMocks/MockCANPGNReceiver.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
Little description and I need to see with you because I don't know how you test with this .h